scripts/Follower.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2
+import rospy
+import math
+from geometry_msgs.msg import PointStamped
+from ackermann_msgs.msg import AckermannDrive
+from std_msgs.msg import String
+from state_controller.msg import TwistLabeled
+from tf import TransformListener
+
+class Follower():
+    def __init__(self):
+        rospy.init_node('Follower', anonymous=True)
+        self.subPoint = rospy.Subscriber('/point2follow', PointStamped, self.callback)
+        self.pubDrive = rospy.Publisher('/state_controller/cmd_behavior_twist',TwistLabeled, queue_size=10)
+        self.point = None
```

Comment:
Maybe make this goalPoint, or something, seems a bit abstract as "point". Docstring could also help.